micromamba.hcl

```diff
@@ -0,0 +1,37 @@
+description = "Micromamba is a tiny version of the mamba package manager which is a CLI tool to manage conda environments."
+binaries = ["micromamba"]
+test = "micromamba --help"
+env = {
+  "MAMBA_ROOT_PREFIX": "${HERMIT_ENV}/.hermit/micromamba",
+}
+
+version "1.5.3-0" {
+  auto-version {
+    github-release = "mamba-org/micromamba-releases"
+  }
+}
+
+linux {
+  source = "https://github.com/mamba-org/micromamba-releases/releases/download/${version}/micromamba-linux-64"
+  on "unpack" {
+    rename {
+      from = "${root}/micromamba-linux-64"
+      to = "${root}/micromamba"
+    }
+  }
+}
+
+darwin {
```

Comment:
It'd be good to have `darwin arm64` here as well, and change this to `darwin amd64`